otx/algorithms/classification/configs/deit_tiny/hpo_config.yaml

```diff
@@ -0,0 +1,15 @@
+metric: accuracy
+search_algorithm: asha
+hp_space:
+  learning_parameters.learning_rate:
+    param_type: qloguniform
+    range:
+      - 0.00098
+      - 0.0245
+      - 0.00001
+  learning_parameters.batch_size:
+    param_type: qloguniform
+    range:
+      - 42
+      - 96
+      - 2
```

Comment:
@eunwoosh  Could you verify whether is it valid configuration or not?

Reply:
I think this is copied from efficientnetb0. Aligning it with Deti hp should be better.
Could you change the range of learning rate as 0.0001 ~ 0.01 and batch size as 4 ~ 16?

Reply:
Thank you for quick check, is below config right?
```
metric: accuracy
search_algorithm: asha
hp_space:
  learning_parameters.learning_rate:
    param_type: qloguniform
    range:
      - 0.0001
      - 0.0010
      - 0.01
  learning_parameters.batch_size:
    param_type: qloguniform
    range:
      - 4
      - 16
      - 2
```

Reply:
change learning rate part as below. Other part seems good.
```
  learning_parameters.learning_rate:
    param_type: qloguniform
    range:
      - 0.0001
      - 0.01
      - 0.0001
```

Reply:
Reflected, thank you!

Reply:
Could you check batch size is also updated?

Reply:
Never mind. I just realized that default batch size is 64 now.

Reply:
I think you changed default learning rate also. Could you update it as below?
```
  learning_parameters.learning_rate:
    param_type: qloguniform
    range:
      - 0.00001
      - 0.001
      - 0.00001
```

Reply:
I'll update soon, thank you for noticing again.

Reply:
Updated in 43ff369d444614dffc3d0f4b8d60fd7d1e8582b2.